admin_guide/managing_projects.adoc

```diff
@@ -0,0 +1,170 @@
+
+====
+----
+...
+projectConfig:
+  nodeSelector: "type=user-node,region=east"
+...
+----
+====
+
+Ensure to restart the master OpenShift service for the changes to take effect:
+
+====
+----
+# systemctl restart atomic-openshift-master
+----
+====
+
+To create an individual project with a node selector, use the `--node-selector` option when creating a project. For example, if you have an OpenShift topology with multiple regions, you can use a node selector to help restrict specific OpenShift projects to only deploy pods onto nodes in a specific region. The following creates a new project named `myproject` and dictates that pods be deployed onto nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project myproject \
+    --node-selector='type=user-node,region=east'
+----
+====
+
+Once this command is run, this becomes the adminstrator-set node selector for all pods contained in the indicated project.
+
+OpenShift developers can set a node selector on their pod configuration if they wish to restrict nodes even further. This will be added to the project node selector, meaning it will be more restrictive.
+
+[NOTE]
+====
+While the `new-project` subcommand is available for both `oadm` and `oc`, the cluster administrator and developer commands respectively, creating a new project with a node selector is only available with the `oadm` command. The `new-project` subcommand is not available to project developers when self-provisioning projects.
+====
+
+Using the `oadm new-project` command adds an `annotation` section to the project configuration. Changing the `openshift.io/node-selector` value overrides the default:
+
+====
+----
+...
+metadata:
+  annotations:
+    openshift.io/node-selector: type=user-node,region=east
+...
+----
+====
+
+If `openshift.io/node-selector` is set to an empty string (`oadm new-project --node-selector=""`), the project will not have an adminstrator-set node selector, even if the cluster-wide default has been set. This means that, as an OpenShift administrator, you can set a default to restrict developer projects to a subset of nodes and still enable infrastructure or other projects to schedule the entire cluster.
+
+However, you can still dictate node selector values for all projects with a node selector value. If a project has been created with the above annotation (`openshift.io/node-selector: type=user-node,region=east`) and a developer sets another node selector on a pod in that project, for example `clearance=classified`, the pod will only ever be scheduled on nodes that have all three labels (`type=user-node`, `region=east` and `clearance=classified`). If they decided they wanted to schedule on `region=west` and set a pod node selector accordingly, they would be demanding nodes with labels `region=east` and `region=west`, which cannot work and will never be scheduled, because labels can only be set to one value. Thus the developer can, per-pod, augment (but not override) the project node selector.
```

Comment:
"However" seems like a non-sequitur here. How about (combining the intro from above):

> OpenShift developers can set a node selector on their pod configuration if they wish to restrict nodes even further. This will be added to the project node selector, meaning it will be more restrictive. Therefore, the administrator can still dictate node selector values for all projects with a node selector value.
> 
> As an example, if a project has been created...